includes/acl/compression/impl/write_stream_data.h

```diff
@@ -331,8 +331,19 @@ namespace acl
 				}
 				else
 				{
+
+#ifdef ACL_BIT_RATE_EXPANSION
+
+					const uint64_t* raw_sample_u64 = safe_ptr_cast<const uint64_t>(raw_sample_ptr);
```

Comment:
Aren't these equivalent?

Reply:
Not quite.  This is the change I tracked down to fix high bitrate regression failures in 1.3.
`const uint64_t raw_sample_u64 = *safe_ptr_cast<const uint64_t>(raw_sample_ptr);`
This copies 64 bits of memory into a local 64 bit value.  Then the address of that local value is passed to the next stage.  This only works if num_bits_at_bit_rate <= 64, otherwise the last few bits are junk.